.github/workflows/network-operator-docker-and-helm-ci.yaml

```diff
@@ -0,0 +1,85 @@
+name: Network Operator Docker and Helm CI
+
+on:
+  push:
+    branches:
+    - "master"
+    - "v[0-9]+.[0-9]+.x"
+
+env:
+  REGISTRY: nvcr.io/nvstaging/mellanox  # for docker
```

Comment:
Although I agree that this (and several other) variables could be renamed to be more informative and accurate, this would require changing also in the Makefile. Such a change would prevent the option of rolling back, in case any issue rises with the refactored CI.
In a future task, I'd be happy to refactor the Makefile (and adjust the CI appropriately).